resources/ui-app/src/routes/example-page.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'react-router-dom/Link';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
```

Comment:
Can we avoid usage of `@folio/react-intl` in advance of `react-intl` package?  Moreover I would opt to use `FormattedMessage` instead of `FormattedHTMLMessage` according to note in theirs documentation. 

`<FormattedHTMLMessage>`
`Note: This component is provided for apps that have legacy external strings which contain HTML, but is not recommended, use <FormattedMessage> instead, if you can.`